src/Models/Schedule.cs

```diff
@@ -15,4 +17,28 @@ public class Schedule
     /// The number of innings scheduled for the game. 
     /// </summary>
     public int? ScheduledInnings { get; set; }
+
+    /// <summary>
+    /// The status of the game.  A few possible codes are "F" for Final, "I" for In Progress, 
+    /// "IR" for Delayed, "P" for Pre-game, and "S" for Sceduled.
+    /// </summary>
+    public GameStatus? StatusCode { get; set; }
+
+    /// <summary>
+    /// Gets the GameStatus enum value based on the provided status code string.
+    /// </summary>
+    /// <param name="statusCode">The status code string.</param>
+    /// <returns>The corresponding GameStatus enum value, or null if the status code is unknown.</returns>
+    public static GameStatus? GetStatusCode(string? statusCode)
```

Comment:
I thought about adding a Utils folder for functions like this, but I didn't want to encroach on the existing folder structure.  I'm happy to put this function wherever.